src/wallet/sqlite.h

```diff
@@ -83,6 +92,8 @@ class SQLiteDatabase : public WalletDatabase
 
     /** Make a SQLiteBatch connected to this database */
     std::unique_ptr<DatabaseBatch> MakeBatch(const char* mode = "r+", bool flush_on_close = true) override;
+
+    sqlite3* m_db;
```

Comment:
In commit "Constructors, destructors, and relevant private fields for SQLiteDatabase/Batch" (08847ca4de58c112da4b6455230f7b454afa543c)

Minor: Suggest `sqlite3* m_db{nullptr};` or `sqlite3* m_db = nullptr;` here to be sure this is safe without looking and even if someone adds another constructor.

Reply:
Done